sourcery_analytics/settings.py

```diff
@@ -1,24 +1,37 @@
+"""Models describing sourcery-analytics settings."""
 import pathlib
 
 import pydantic
 import tomli
 
 
 class ThresholdSettings(pydantic.BaseModel):
+    """Model describing the available thresholds and their defaults."""
+
     method_length: pydantic.PositiveInt = 15
     method_cyclomatic_complexity: pydantic.PositiveInt = 10
     method_cognitive_complexity: pydantic.PositiveInt = 10
     method_working_memory: pydantic.PositiveInt = 20
 
-    class Config:
-        extras_allowed = True
-
 
 class Settings(pydantic.BaseSettings):
+    """Model describing general sourcery-analytics settings and their construction."""
```

Comment:
**suggestion (docstrings):** Please update the docstring for function: `Settings`

Reason for update: The docstring was added to the class.

Suggested new docstring:
```
"""Model describing general sourcery-analytics settings and their construction."""
```